src/tools/bin/tbg

```diff
@@ -423,8 +427,12 @@ fi
 start_dir=`dirname $0`
 
 if [ -d "$job_outDir" ] ; then
-    echo "job name already in use, can't create new folder"
-    exit 1
+	if [ "$force" = "true" ] ; then
+		echo "Warning: using existing folder for job on user-request ( -f )"
```

Comment:
maybe remove *for job*

Reply:
maybe no spaces around the `-f`, also no `-` in *user request*:
`... user request (-f)`

Reply:
I removed the spaces, but left the '-'. In exchange I swapped the parenthesis with square brackets to achieve some consistency with the help message.